src/Components/Promises/PromiseTable/index.js

```diff
@@ -53,6 +71,23 @@ export class PromiseTable extends React.Component {
     }
   };
 
+  renderSubjects = (subjects?: Array<DecoratedSubject>) => {
+    if (!subjects) return null;
+    const { unlinkSubjectPromise } = this.props;
+    return subjects.map(
+      subject =>
+        subject
```

Comment:
is it possible for `subject`  to be falsy here? Wondering if that logic can be removed and just mapped directly to `Chip`

Reply:
You're correct. There is no need to do a null check here